src/main/java/seedu/address/logic/commands/UnmarkAttendanceByStudentCommand.java

```diff
@@ -0,0 +1,111 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ATTENDANCE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TUTORIAL;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.participation.Participation;
+import seedu.address.model.person.Attendance;
+import seedu.address.model.person.Person;
+import seedu.address.model.tutorial.Tutorial;
+
+/**
+ * Marks the attendance of a person identified using it's displayed index from the addres book as absent.
+ */
+public class UnmarkAttendanceByStudentCommand extends Command {
+
+    public static final String COMMAND_WORD = "umas";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Unmarks the attendance of the student identified "
+            + "by the index number used in the displayed person list.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_ATTENDANCE + "ATTENDANCE]"
```

Comment:
Think by UG standard, including square bracket refers to the field being optional(eg. inside editCommand)
Not too sure if we are going to apply this standard here also, we can discuss about it

Reply:
Good observation, was following the standard for the other commands relating to attend, but will edit this for for all them. Thanks!